lib/board.rb

```diff
@@ -0,0 +1,63 @@
+class Board
+
+attr_accessor :cells
+
+  #sets array w/ 9 slots & '' on each slot
+  def initialize
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.